CHANGELOG.md

```diff
@@ -40,22 +40,28 @@ Ref: https://keepachangelog.com/en/1.0.0/
 
 * __Modules__
   * (x/staking) [\#7499](https://github.com/cosmos/cosmos-sdk/pull/7499) `BondStatus` is now a protobuf `enum` instead of an `int32`, and JSON serialized using its protobuf name, so expect names like `BOND_STATUS_UNBONDING` as opposed to `Unbonding`.
+  * (x/evidence) [\#7538](https://github.com/cosmos/cosmos-sdk/pull/7538) The ABCI's `Result.Data` field of `MsgSubmitEvidence` does not contain the raw evidence's hash, but the encoded `MsgSubmitEvidenceResponse` struct.
```

Comment:
There's a few other non-nil response types. Should we be adding changelog entries for those as well?

- `MsgBeginRedelegateResponse`
- `MsgUndelegateResponse`
- `MsgSubmitProposalResponse`

Also, just to confirm- does this change the actual response type for CLI / legacy REST interactions? Or just for the new proto interfaces?

Reply:
Yes we should add change log entries. Yes it does affect those but that was all at the binary level anyway

Reply:
Done!